dali/operators/coord/coord_flip.cc

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "dali/operators/coord/coord_flip.h"
+
+namespace dali {
+
+DALI_SCHEMA(CoordFlip)
+    .DocStr(
+        R"code(Transforms normalized coordinates (range [0.0, 1.0]) so that they map to the same place after
+horizontal or/and vertical flip of the input they refer to.)code")
+    .NumInput(1)
+    .NumOutput(1)
+    .AddOptionalArg<TensorLayout>(
+      "layout",
+      R"code(Determines the layout of the coordinates.
+  Possible values are:
+
+  ``x`` (horizontal position), ``y`` (vertical position), ``z`` (depthwise position),
+
+Note: If left empty, ``"xy"`` or ``"xyz"`` will be assumed, depending on the number of dimensions.
+)code",
+      TensorLayout{""})
+    .AddOptionalArg("horizontal", R"code(Perform flip along horizontal axis.)code", 1, true)
+    .AddOptionalArg("vertical", R"code(Perform flip along vertical axis.)code", 0, true)
+    .AddOptionalArg("depthwise", R"code(Perform flip along depthwise axis.)code", 0, true);
```

Comment:
I think, that image flip is (unfortunately) defined the opposite. `horizontal` denotes flip along vertical axis. Maybe we should unify it?

Reply:
Yes, at least per docs: `horizontal (int, optional, default = 1) – Perform a horizontal flip.`

On the other hand BbFlip does the same as this operator, we're already not consistent.

Reply:
Dayum :( Let's discuss what to do with it